application/helpers/remotecontrol/remotecontrol_handle.php

```diff
@@ -288,6 +428,24 @@ public function import_survey($sSessionKey, $sImportData, $sImportDataType, $sNe
      * @param string $sNewname name of the new survey
      * @return array On success: new $iSurveyID in array['newsid']. On failure array with error information
      * */
+    function addToZip($zip, $name, $full_name)
+     /**
+	 * CODE DUPLICATION! This is also part of export.php
+	 * Sorry I can't solve this any better...
+	 * @param PclZip $zip
+	 * @param string $name
+	 * @param string $full_name
+	 */
+	{
+		$zip->add(
+		array(
+		array(
+		PCLZIP_ATT_FILE_NAME => $name,
+		PCLZIP_ATT_FILE_NEW_FULL_NAME => $full_name
+		)
+		)
+		);
+	}
```

Comment:
This is probably not the right place for this since it's between the `copy_survey` function and its "docstring"

Reply:
Hi @edgarrmondragon 
could you suggest a better location ?

Reply:
@AmauryVanEspen Probably above line 422